Project-Hotpot/Project-Hotpot/Parse-Backend/PFPlaylist.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  PFPlaylist.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/18/22.
+//
+
+import Foundation
+import Parse
+import AFNetworking
+
+class PFPlaylist: PFObject, PFSubclassing {
+    @NSManaged var user : PFUser
+    @NSManaged var name : String?
+    @NSManaged var songArray : [PFSong]?
+    
+    static func parseClassName() -> String {
+        return "Playlist"
+    }
+    
+    class func createPlaylist(name:String, completion: (Bool) -> Void){
+        let newPlaylist = PFPlaylist()
+        if(PFUser.current() == nil){
+            NSLog("current user is nil")
+            completion(false)
+        }
+        newPlaylist.user = PFUser.current()!
```

Comment:
Swift has a very cool `guard` keyword (https://docs.swift.org/swift-book/ReferenceManual/Statements.html#grammar_guard-statement).
You can use this like `guard let user = PFUser.current() else { /* abort */ } ... newPlaylist.user = user` to guard against missing/incorrect values.
Another nice bit about `guard` is that the compiler will force you to include a return statement. Note that, as you've currently written this, there's no early return here! You're firing off the completion, but still falling through to the rest of the code. This, in combination with the dangerous `!` coalescing operator, means you're likely to crash rather than gracefully handling the failure here.